src/mlang/mpp_ir/mpp_ir_to_bir.ml

```diff
@@ -342,7 +342,7 @@ and translate_mpp_stmt (mpp_program : Mpp_ir.mpp_compute list)
       translate_mpp_function mpp_program m_program
         (List.find (fun decl -> decl.Mpp_ir.name = f) mpp_program)
         real_args ctx
-  | Mpp_ir.Expr (Call (Program, args), _) ->
+  | Mpp_ir.Expr (Call (Program _chain, args), _) ->
```

Comment:
Ok this means that for now we don't do anything with the chain information and we just call all the rules ?

Reply:
Yup. Keryan needs this in his branch.